api/src/plugins/openid-connect/getConfig.js

```diff
@@ -72,6 +72,14 @@ module.exports = options => {
     subjectTypes: ['public', 'pairwise'],
     pairwiseSalt: options.pairwiseSalt,
     interactionUrl: async (ctx, interaction) => `/interaction/${ctx.oidc.uuid}`,
-    scopes: ['admin', 'superadmin'],
-  }
-}
+    scopes: ['admin', 'superadmin', 'offline_access'],
+    ttl: {
+      AccessToken: parseInt(process.env.ACCESS_TOKEN_EXP, 10) || 3600,
```

Comment:
instead of getting these directly from the environment please use the `options` object and pass these in when initializing the plugin